Gimbalize/src/hardwareReader.cpp

```diff
@@ -108,21 +128,41 @@ void loop() {
   Serial.print(altitude);
   Serial.println(" m");
 
+  try {
+    desired_time = millis()
+    std::array<double, 3> BMP_interpolated_values = bmpTmeSync.getValuesAt(desired_time);
+    std::array<double, 3> MPU_interpolated_values = mpuTmeSync.getValuesAt(desired_time);
```

Comment:
"mpuTmeSync" undefined

Reply:
You can also store millis() in a variable at beginning of loop and compare with last loop time (mb realized that cell timing wasn't that accurate)